File: ts/project/.sdk/tm/java/utility/PrepareBody.java
package JAVAPACKAGE.utility;

import JAVAPACKAGE.core.Context;

final class PrepareBody {

  private PrepareBody() {}

  static Object prepareBody(Context ctx) {
    if ("data".equals(ctx.op.input)) {
      return ctx.utility.transformRequest.apply(ctx);
    }

    return null;
  }
}

File: ts/project/.sdk/tm/csharp/utility/PrepareBody.cs
// ProjectName SDK utility: prepareBody.

namespace ProjectNameSdk.Util;

public static partial class SdkUtility
{
    internal static object? PrepareBodyUtil(Context ctx)
    {
        var op = ctx.Op!;

        if (op.Input == "data")
        {
            var body = ctx.Utility!.TransformRequest(ctx);
            return body;
        }

        return null;
    }
}
